chia/consensus/multiprocess_validation.py

```diff
@@ -5,6 +5,8 @@
 from dataclasses import dataclass
 from typing import Dict, List, Optional, Sequence, Tuple, Union, Callable
 
+from chia.util import cached_bls
```

Comment:
my understanding is that the caching saves duplicate work between validating mempool entries and full blocks with those entries in them. while syncing, wouldn't the cache just slow us down?

Reply:
Yes you are right. We would compute each pairing individually and add them to the cache. If we avoid using cache here, we can batch verify which computes all the pairings together and is more efficient. I will make this change.